bin/shutdown.pid

```diff
@@ -0,0 +1 @@
+14874
```

Comment:
Please remove this file.

Reply:
This was solved in df5aa7d3bda8d19bf734ebaf2e7dc505a5f60ac3.